apps/accounts/forms/auth.py

```diff
@@ -296,7 +320,10 @@ class UsernameEmailAuthenticationForm(forms.Form):
     recaptcha = forms.CharField(required=False, widget=forms.HiddenInput)
 
     error_messages = {
-        'invalid_login': _('Please enter a correct %(username_email)s and %(password)s. Note that password is case-sensitive.'),
+        'invalid_login': _(
+            'Please enter a correct %(username_email)s and %(password)s. '
```

Comment:
let's keep what we had before and add `# noqa`

Reply:
Done